docs/words.txt

```diff
@@ -1,17 +1,28 @@
 aggregator
 app
+backend
 config
 Config
 configurables
 Cookiecutter
+crypto
 Einhorn
+gevent
 Gevent
+healthcheck
 hostname
 INI
 lifecycle
 namespace
+predecided
 pubsub
+Queing
```

Comment:
http://english.stackexchange.com/questions/60852/queueing-or-queuing 

I can switch to the long string of vowels form if you think it'd be more standard though.

This file is a custom dictionary for the sphinx spellchecker: http://sphinxcontrib-spelling.readthedocs.org/en/latest/